lib/cypress-rails/config.rb

```diff
@@ -2,17 +2,19 @@
 
 module CypressRails
   class Config
-    attr_accessor :dir, :host, :port, :base_path, :transactional_server, :cypress_cli_opts
+    attr_accessor :dir, :cy_dir, :host, :port, :base_path, :transactional_server, :cypress_cli_opts
```

Comment:
I think I'd rather spell these out as :rails_dir and :cypress_dir for clarity

Reply:
Makes sense! Will add!